consistency_fail.gemspec

```diff
@@ -21,9 +21,10 @@ stop ignoring the C in ACID.
 EOF
   s.license = "MIT"
 
-  s.add_development_dependency "activerecord", "~>5.0"
+  s.add_development_dependency "activerecord", "~>6.0.3.4"
```

Comment:
I'd say we need to relax this condition to `~> 6.0` so we don't need to update the gem every time Rails release a minor update.